tasks/24_index-to-insert/index-to-insert.js

```diff
@@ -0,0 +1,6 @@
+export function getIndexToInsert(arr, num) {
+  const result = arr.slice();
```

Comment:
You could fo it in more shoer way like [...arr, num]

Reply:
i tried it, it's really better to read, thanks